src/gui/BandSplitter/BandSplitterPlot.h

```diff
@@ -25,6 +25,7 @@ class BandSplitterPlot : public chowdsp::EQ::EqualizerPlot
     void updateCutoffFrequency();
     void updateFilterSlope();
     void updateSpectrumPlots();
+    std::function<const chowdsp::FreqHzParameter::Ptr&(int)> getCutoffParam;
```

Comment:
I guess we could just make this a member function (maybe a static member function?) That would probably be a little bit simpler :).